src/Mod/Sketcher/App/SketchObject.cpp

```diff
@@ -3415,6 +3415,19 @@ int SketchObject::addSymmetric(const std::vector<int> &geoIdList, int refGeoId,
     std::map<int, int> geoIdMap;
     std::map<int, bool> isStartEndInverted;
 
+    // Find out if reference is aligned with V or H axis,
+    // if so we can keep Vertical and Horizontal constrants in the mirrored geometry.
+    bool refIsAxisAligned = false;
+    if (refGeoId == Sketcher::GeoEnum::VAxis || refGeoId == Sketcher::GeoEnum::HAxis)

```

Comment:
I would prefer an open brace at the end of 3421 and a closed brace before line 3423. The reason is that it is usually easier not the get lost when the else part is multi-line and uses braces too. But this is really pure stylistic.

Reply:
Sure